vesta/chars.py

```diff
@@ -161,18 +161,18 @@ def encode_text(
     max_cols = COLS - margin * 2
     rows: List[List[int]] = []
 
-    def find_break(line: List[int]) -> int:
+    def find_break(line: list[int]) -> tuple[int, bool]:
```

Comment:
We need to continue using the `typing.` names until we drop support for versions earlier than Python 3.9 ([PEP 585](https://www.python.org/dev/peps/pep-0585/)).

Reply:
I forgot to check for the `from __future__ import annotations` at the top. Would you rather I add that or use the from typing imports?

Reply:
Let's use the `typing` imports for this change. Now that we're Python 3.7+, we can separately consider using `__future__.annotations` in its own change.